src/parse.rs

```diff
@@ -319,15 +372,41 @@ fn gen_bytecode(parse_tree: &Node<u16>, grm: &YaccGrammar, input: &str) -> Bytec
                                 let int = lit_value.parse::<i32>().unwrap();
                                 ctx.gen_bc(Instr::PUSH_INT(int))
                             }
+                            "STR_LITERAL" => {
+                                ctx.gen_bc(Instr::PUSH_STR(lit_value))
+                            }
                             _ => panic!("NotYetImplemented")
-                        }
+                        };
                     }
                     _ => panic!("unknown expression")
                 }
             }
         }
     }
 
+    // arg_list_opt :
+    //              | arg_list
+    //              ;
+
+    // arg_list : expression
+    //          | parameter_list "COMMA" expression
+    //          ;
+    fn gen_args(node: &Node<u16>, ctx: &mut CompilerContext) {
+        match *node {
```

Comment:
This would be clearer as `if let Node::Nonterm{ nonterm_idx, ref nodes } = *node { ... }` I think.